app/controllers/concerns/admin/participants/find_induction_records.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Admin
+  module Participants
+    module FindInductionRecords
+      extend ActiveSupport::Concern
+
+      def latest_induction_record
+        induction_records.first
+      end
+
+      def historical_induction_records
+        induction_records[1..]
+      end
+
+      def all_induction_records
+        induction_records
+      end
+
+    private
+
+      def induction_records
+        @induction_records ||= @participant_profile
+          .induction_records
+          .eager_load(
+            :appropriate_body,
+            :preferred_identity,
+            :schedule,
+            induction_programme: {
+              partnership: :lead_provider,
+              school_cohort: %i[cohort school],
+            },
+            mentor_profile: :user,
+          )
+          .order(created_at: :desc)
```

Comment:
We sometimes have issues with this ordering, I'm not sure why `created_at` is used, I think it would be better to do `order(start_date: :desc, created_at: :desc)` to prevent records appearing wrongly if they've been created out of order / de-duped / merged etc.

Reply:
I think that ideally the order should only be `created_at: :desc` but, as Tony mentions, we still have issues with the start-date/end-date of records not matching the creation order.

If we finally order the records according to Tony's suggestion, can we please add the creation date to the list of fields displayed for each induction record in the `induction records` tab? That would help to investigate and solve Zendesk tickets.

Reply:
Thank you, made changes addressing both requests.